_includes/tutorials/aggregating-count/flinksql/code/docker-compose.yml

```diff
@@ -79,3 +79,28 @@ services:
         FLINK_PROPERTIES=
         jobmanager.rpc.address: flink-jobmanager
         taskmanager.numberOfTaskSlots: 10
+
+  rest-proxy:
+    image: confluentinc/cp-kafka-rest:6.0.14
+    depends_on:
+      - zookeeper
+      - broker
+      - schema-registry
+    ports:
+      - "8082:8082"
+    hostname: rest-proxy
+    container_name: rest-proxy
+    environment:
+      KAFKA_REST_HOST_NAME: rest-proxy
+      KAFKA_REST_BOOTSTRAP_SERVERS: 'broker:9092'
+      KAFKA_REST_LISTENERS: "http://0.0.0.0:8082"
+      KAFKA_REST_SCHEMA_REGISTRY_URL: 'http://schema-registry:8081'
```

Comment:
is 8081 the default port for sr?

Reply:
> is 8081 the default port for sr?

Yep, normally 8081 is for Schema Registry, 8082 is for Kafka REST. I don't recall seeing 8085 being used for anything though.

Reply:
Here's some example documentation: https://docs.confluent.io/platform/current/schema-registry/develop/using.html